docker/server/configs/default.conf

```diff
@@ -0,0 +1,49 @@
+server {
+    listen 80;
+    server_name _;
+
+    root /var/www/html;
+
+    index index.html index.htm index.php;
+
+    # server_name _;
+
+    charset utf-8;
+
+    location = /favicon.ico { log_not_found off; access_log off; }
+    location = /robots.txt  { log_not_found off; access_log off; }
+
+    location /files {
+        try_files $uri $uri/ /server.php?$query_string;
+    }
+
+#    access_log /var/log/nginx/tus.server-access.log nginx;
+#    error_log  /var/log/nginx/tus.server-error.log warn;
```

Comment:
forgot to uncomment? :)

Reply:
These are not needed anymore thus I have commented them. Server logs are mounted in folder `docker/.data/server/logs` so if any error happens we can look at file `docker/.data/server/logs/error.log`. I will remove these commented line from `default.conf`